src/components/Article.jsx

```diff
@@ -2,12 +2,19 @@ import axios from "axios";
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import formatDate from "../assets/utils";
+import PostComment from "./PostComment";
+import hardcodedUser from "../assets/hardcodedUser";
 
 function Article({ users }) {
```

Comment:
your code looks good but this component has become quite large! consider breaking this down into smaller components where you can handle the logic of the comments, posting and deleting them separately to reduce code bloat!